### DISCUSSION TITLE
How to open pixi js app without server?

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7690] [Q&A]
I build my pixi app via 'npm run build' in Vite.
I want to open it just as clicking index.html file withot running server.
Is it possible?
Currently I'm getting this error message.
![image](https://user-images.githubusercontent.com/87828192/128798152-f84a274d-c430-4016-bda6-845e32fa4633.png)

**Comment by bigtimebuddy** [comment]
Short answer is not really. There are however a few very easy web-servers you can use to run. Try npmjs.com/http-server, which is dead simple to run.

**Comment by ivanpopelyshev** [comment]
dont use absolute paths then, only relative, so no "/assets". Also, loader /Texture.from() can only take image files , no json for you! Also, forget about readPixels() and getImageData() (extract!) if you use it.

The usual for any html5 canvas app

Actually, are you sure that Vite can work through this `file:///` environment? try post this in Vite github too :)
